helm/aws-operator-chart/templates/03-configmap.yaml

```diff
@@ -11,6 +11,9 @@ data:
     service:
       aws:
         region: '{{ .Values.Installation.V1.Provider.AWS.Region }}'
+      guest:
+        update:
+          enabled: {{ .Values.Installation.V1.Guest.Update.Enabled }}
```

Comment:
We use this for KVM already so installations are prepared and we can safely merge.